packages/mobile/package.json

```diff
@@ -112,6 +112,7 @@
     "react-native-draggable-flatlist": "3.0.7",
     "react-native-dynamic": "1.0.0",
     "react-native-fast-crypto": "2.2.0",
+    "react-native-fast-image": "8.6.3",
```

Comment:
Let's keep an eye on our android crash logs when this goes out. I have a feeling we might see some issues

Reply:
sounds like a plannn